.github/workflows/github-actions-release-candidate.yml

```diff
@@ -30,8 +30,8 @@ jobs:
         with:
           bundler-cache: true
       - name: Python Setup
-        uses: actions/setup-python@v4 
-        with: 
+        uses: actions/setup-python@v4
+        with:
```

Comment:
This whole file is adding spaces. 

Can you just run a `git checkout origin/master -- .github/workflows/github-actions-release-candidate.yml` on this branch. That will reset it to what's on master, so that these syntax changes don't occur.